### ISSUE TITLE
Should I scale up the learning rate with batch size?

### ISSUE THREAD

**Issue by SwayStar123** [lllyasviel/ControlNet#322]
If I can afford to train at batch size of 64 with optional gradient accumulation, should I increase the loss in accordance?
